bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -193,17 +204,6 @@ private HBaseRequestAdapter createAdapter(TableName tableName) {
     return new HBaseRequestAdapter(options.getClusterName(), tableName, conf);
   }
 
-  @Override
-  public BufferedMutator getBufferedMutator(TableName tableName) throws IOException {
```

Comment:
nit: This seems to have just moved around, causing a diff, was there a reason for the move?